schema.graphql

```diff
@@ -993,6 +993,8 @@ input UpdateOrganizationInput {
 }
```

Comment:
Also revert any change you make in this file, here you should only open a PR with changes in your `src/typeDefs/input.ts` file otherwise one of the workflow tests will fail because of `unauthorised file changes`

Reply:
Done